lib/kuniri/language/ruby/attribute_ruby.rb

```diff
@@ -70,54 +70,50 @@ def prepare_final_string(pString)
       end
 
       # Override
-      def handle_multiple_declaration_with_comma(pString)
-        listOfAttributes = []
-        pString = pString.split(",")
-        pString.each do |variable|
-          return nil if variable.scan(/=/).count > 1
+      def handle_line(pString)
```

Comment:
I recommend you put this function at the end of the code in a private section, because handle_line isn't on parent class.